ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/MiniProtocol/ChainSync/Client/Jumping.hs

```diff
@@ -0,0 +1,828 @@
+{-# LANGUAGE DeriveAnyClass       #-}
+{-# LANGUAGE DeriveGeneric        #-}
+{-# LANGUAGE DerivingStrategies   #-}
+{-# LANGUAGE FlexibleContexts     #-}
+{-# LANGUAGE FlexibleInstances    #-}
+{-# LANGUAGE LambdaCase           #-}
+{-# LANGUAGE NamedFieldPuns       #-}
+{-# LANGUAGE RankNTypes           #-}
+{-# LANGUAGE ScopedTypeVariables  #-}
+{-# LANGUAGE StandaloneDeriving   #-}
+{-# LANGUAGE TupleSections        #-}
+{-# LANGUAGE TypeApplications     #-}
+{-# LANGUAGE UndecidableInstances #-}
+
+-- | ChainSync jumping (CSJ) is an optimization for the ChainSync protocol that
+-- allows nodes to sync without downloading headers from all of the honest
+-- peers. This load is undesirable as it slows down all the peers involved.
+--
+-- The idea is to download the headers of a chain from a single peer (the
+-- dynamo) and then ask periodically to the other peers (the jumpers) whether
+-- they agree with the dynamo's chain.
+--
+-- When the jumpers disagree with the dynamo, the jumper with the oldest
```

Comment:
```suggestion
-- When the jumpers disagree with the dynamo (therefore becoming an objector), the jumper with the oldest
```

You refer to the objector two lines below but it was not mentioned who that was

Reply:
A dissenting jumper might not become an objector if there is a dissenting jumper with an earlier intersection. But the fix is close to what you are suggesting. :+1:

Reply:
Addressed in 90ad07c.